docs/day2/05_scale.md

```diff
@@ -110,18 +127,14 @@ allows the number of CPUs the process' task should use.
 The `FASTQC` tasks processes paired reads (2 files) per task. Adding
 `-t ${task.cpus}` allows them to be processed simultaneously.  
 
-Run again:  
+Update your configuration file:  
 
-```bash
-nextflow run main.nf -with-report fastqc_mt.html
+```groovy linenums="1" title="nextflow.config
+process.cpus = 2
+docker.enabled = true
 ```
 
-> Compare `cpus_2.html` and `fastqc_mt.html` and note the differences between
-> %cpu and duration of the FASTQC tasks
-
-- run time decreased  
-- %cpu increased  
-- duration decreased  
+> How can we demonstrate the benefit here, without profile/trace  
```

Comment:
Here we can show them what happens when you add trace, report, timeline to the `nextflow.config`. This will give them insight into task resource requirements and consumption.